metaflow/plugins/conda/batch_bootstrap.py

```diff
@@ -49,11 +49,8 @@ def download_conda_packages(flow_name, env_id):
 def install_conda_environment(env_id, packages):
     args = [
         "if ! type conda  >/dev/null 2>&1; \
-            then wget --no-check-certificate https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-linux-64.exe -O conda >/dev/null 2>&1; \
-            chmod +x conda; \
-            export PATH=$PATH:{0}; fi".format(
-            os.getcwd()
-        ),
+            then wget --no-check-certificate https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O Miniforge3.sh >/dev/null 2>&1; \
+            bash ./Miniforge3.sh -b; ln -sf /root/miniforge3/bin/conda /usr/local/bin/conda ; fi",
```

Comment:
@oavdeev Would we be able to always execute `ln -sf /root/miniforge3/bin/conda /usr/local/bin/conda`?

Reply:
Also, it seems that Miniforge3 brings in Python3.9 with it as well, which is pure bloat for us. Should we look into instead using micromamba `wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/conda`

Reply:
For `ln -sf`, it prob won't work in a non-root container, but the previous version wouldn't work either.

No opinion on micromamba, I'd just pick whatever distribution seems to be the most mainstream so we don't have to yank it out again in a year.

Reply:
I gave micromamba a try, unfortunately it looks like it wouldn't work as a drop in replacement. For one thing, it doesn't seem to support `CONDA_JSON` flag used [here](https://github.com/Netflix/metaflow/blob/switch-to-miniforge/metaflow/plugins/conda/conda.py#L174:L174) (i didn't look further)

Reply:
And I did a little more investigating on non-root, it is a bit finicky. It seems to generate conda entrypoint differently when installer is run as non-root which may cause it to use the wrong python interpreter. Someone would need to look further into that; i'd propose we do this in as a separate ticket and PR.